app/src/main/java/com/rkm/tasky/network/model/response/ReminderResponse.kt

```diff
@@ -0,0 +1,16 @@
+package com.rkm.tasky.network.model.response
+
+import com.google.gson.annotations.SerializedName
+
+data class ReminderResponse(
+    @SerializedName("id")
+    val id: String,
+    @SerializedName("title")
+    val title: String,
+    @SerializedName("description")
+    val description: String?,
+    @SerializedName("time")
+    val time: Long,
+    @SerializedName("remindAt")
+    val remindAt: Long
+)
```

Comment:
What's the difference between this class and `ReminderDTO`? These are in fact identical except for the JSON field names, so I don't really see the point in duplicating them. Technically, anything you send or get via the network is a DTO